conduct.md

```diff
@@ -53,11 +53,16 @@ Allegheny's honor code for all submitted work.
 
 * Acceptable Conduct (A):
   * Members shall
-    * :heavy_exclamation_mark: TODO: add items in the A category for deadlines.
+    * Turn assignments in on the due date, by the time set by JJ; the only exception to this rule is if the student is excused by the professor.
```

Comment:
It would be great if you can split this bullet point into two smaller ones. Restructuring the statement can also give it more clarity

Reply:
We edited the way it's phrased, so it reads more clearly now.